apps/developers/package.json

```diff
@@ -5,7 +5,7 @@
   "scripts": {
     "docusaurus": "docusaurus",
     "start": "docusaurus start",
-    "build": "docusaurus build",
+    "--build": "docusaurus build",
```

Comment:
Why this change?

Reply:
To disable it, there seems to be some issues with the Docusaurus build but as we're not currently using this, I don't think it's worth spending time fixing.